tests/utilities/test_attributes_sim2seis.py

```diff
@@ -0,0 +1,144 @@
+import logging
+
+import numpy as np
+import pytest
+import xtgeo
+
+from fmu.tools import sample_attributes_for_sim2seis
+from fmu.tools.utilities.sample_attributes import Attrs, _get_layer
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope="module", name="local_testdata")
+def fixture_testdata(testdata_path):
+    """Read test data from file as fixture."""
+    gpath = testdata_path / "3dgrids/drogon/4/simgrid_with_region_zone_attr.roff"
+
+    grid = xtgeo.grid_from_file(gpath)
+    region = xtgeo.gridproperty_from_file(gpath, name="Region")
+    attr = xtgeo.gridproperty_from_file(gpath, name="MyAttr")
+    zone = xtgeo.gridproperty_from_file(gpath, name="Zone")
+
+    attr_surface = xtgeo.surface_from_grid3d(grid, property=attr, rfactor=4)
+
+    return grid, region, zone, attr_surface
+
+
+@pytest.mark.parametrize(
+    "where, expected",
+    [
+        (("Valysar", "top"), 1),
+        (("Valysar", "center"), 6),
+        (("Valysar", "base"), 10),
+        (("Therys", "top"), 11),
+        (("Therys", "center"), 19),
+        (("Therys", "base"), 27),
+        (("Volon", "top"), 28),
+        (("Volon", "center"), 30),
+        (("Volon", "base"), 32),
+        (("", "top"), 1),
+        (("", "center"), 16),
+        (("", "base"), 32),
+    ],
+)
+def test_getlayer_function(local_testdata, where, expected):
+    """Test the internal _get_layer() function (where layer number has base 1)."""
+
+    grid, _, zone, _ = local_testdata
+
+    assert _get_layer(where, grid, zone) == expected
+
+
+def test_attr_maps_sim2seis(local_testdata):
+    """Test a basic case."""
+
+    grid, region, zone, attr_surface = local_testdata
+
+    attr_surface_error = attr_surface * 0.2
+    attr_surface_error.values = np.abs(attr_surface_error.values)
+
+    df = sample_attributes_for_sim2seis(
+        grid,
+        attr_surface,
+        attribute_error=attr_surface_error,
+        region=region,
+        position=("Valysar", "top"),
+        zone=zone,
+        debug=False,  # this is a 'developer' kwargs to the function
+    )
+    assert Attrs.OBS.value in df.columns
+    assert Attrs.OBS_ERROR.value in df.columns
+    assert np.mean(df.OBS.to_numpy()) == pytest.approx(
+        attr_surface.values.mean(), rel=0.05
+    )
+    assert np.max(df.REGION.to_numpy()) == 7.0
```

Comment:
Consider using built in pandas method here for readability 

`df["REGION"].max()`